HIRS_Utils/src/main/java/hirs/utils/HexUtils.java

```diff
@@ -74,4 +76,41 @@ public static Integer hexToInt(final String s) {
         System.arraycopy(b, start, copy, 0, end - start + 1);
         return copy;
     }
+
+    /**
+     * Takes in a byte array and reverses the order.
+     * @param in  byte array to reverse
+     * @return reversed byte array
+     */
+   public static byte[] leReverseByte(final byte[] in) {
```

Comment:
You can use ArrayUtils.reverse(byte[]) or Collections.reverse(List<T>).

Reply:
OK, will replace in a future PR.

Reply:
I added an issue to cover this. #227